### ISSUE TITLE
WMT en-de data preprocessing bug

### ISSUE THREAD

**Issue by YuxianMeng** [pytorch/fairseq#3454]
Hi, thanks for providing scripts to reproduce wmt en-de translation task!

I followed [your scripts](https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-wmt14en2de.sh) to preprocess wmt data and train a transformer model. However, during decoding, I found that `„`  does not appear in dictionay, thus being indexed and translated to `unk`. 

I guess the reason is that you normalize punctuations (transform "„" to "&quot") only in training data, but did not do it in test data.

**Comment by YuxianMeng** [comment]
I found that this issue is caused by both the above reason and "--score_reference", and does not hamper the normal generation.
